juju/juju
# Fix non-microk8s cloud finding.

When microk8s is not installed and you want to bootstrap a non-microk8s cloud, it would error:
```


## QA steps

Uninstall microk8s and install minikube(or another k8s in kubeconfig). Make sure you can at least attempt a bootstrap.

## Documentation changes

N/A

## Bug reference

N/A
